src/button.css.ts

```diff
@@ -421,23 +421,23 @@ const dangerLinkInverse: ComplexStyleRule = [
     link,
     sprinkles({
         color: vars.colors.textLinkDanger,
```

Comment:
@yceballost this should be textLinkDangerInverse according to specs, but that token doesn't exist :(

Reply:
I think textLinkDangerInverse is not needed 🤔